cmd/brew-bootstrap-asdf

```diff
@@ -31,25 +31,21 @@ if ! which asdf &>/dev/null; then
 fi
 
 if [ ! -f .tool-versions ]; then
-  abort "Error: you need a '.tool-versions' file to install packages"
+  abort "Error: you need a '.tool-versions' file to install packages!"
 fi
 
 for plugin in $(cat .tool-versions | awk '{print $1}'); do
   if [ -z $(asdf plugin-list | grep $plugin) ]; then
-    asdf plugin-add $plugin
+    asdf plugin add $plugin
   fi
 done
 
-if [ ! -z "$(grep nodejs .tool-versions)" ]; then
-  brew bundle --file=- <<EOF
-brew "gpg"
-EOF
-
-  bash ~/.asdf/plugins/nodejs/bin/import-release-team-keyring
-fi
-
 asdf install
 
+for tool in $(cat .tool-versions); do
```

Comment:
I _think_ we might not want to do this one - the reason being that if `asdf shell` has set the version of a tool using the environment variable, then when folks update their `.tool-versions` file to bump ruby or postgres or whatever, then the change won't have an effect - they'd need to run `asdf shell ruby --unset` first. My fear is that people won't know that they need to run that unset command, and will get stuck debugging why their `.tool-versions` file isn't working as it should.

Agree this wouldn't help with the postgres install issue though - let's dig into that separately perhaps?

Reply:
Totally understand and I agree. It was a little "iffy" having to set the `shell` in the first place. If we do end up getting a Postgres related error when installing the ruby gems, we should find a suitable place to point out that the user should try the `asdf shell` fix.

I will remove this block.